main.js

```diff
@@ -0,0 +1,71 @@
+//PART 1
+//grab text user input
+//grab name user input
+//put that in posts div
+  //'I like cheese - Posted By: Aaron'
+
+//POSTS
+$('#submit').on('click', function () {
+  var userText = $('#post-text').val();
+  var userName = $('#name').val(); 
+
+ // $('.posts').append('<div class="post">'); ??
+
+  $('.posts').append('<p id="post">' + '<button type="button" class="btn btn-link" id="remove">remove</button>' + '<button type="button" class="btn btn-link" id="comment">comment</button>'  + userText + ' - Posted By: ' + userName +'</p>'); 
+
+  $('.posts').append('<hr>');
+
+}); 
+
+
+//PART 2
+//add remove and comments buttons in front of post
+  // looks like --- 'remove comment I like chees - Posted By: aaron'
+// use this for remove and comment <button type="button" class="btn btn-link">Link</button>
+//add a new <div> with each post and comments inside a <p>???? 
+
+//onClick event listener for the comment and remove
+  //changes 'comment' to 'comments'
+  //opens up comments if there are comments
+    //has a form with posts and name boxes
+    //submit comment button 
+      //onClick event listener that posts a comment 
+  //remove button removes comments
+  
+
+//remove button 
+$('.posts').on('click', 'p', function (e) {
```

Comment:
Instead of the 'p' element reference the div element to remove everything (including the hr!)